ArrCharOps.java

```diff
@@ -37,6 +37,9 @@ public static void println(char[] arr) {
      */
     public static char charAt(char[] arr, int index) {
         // Replace the following statement with your code
+        if (index <= arr.length) {
+            return arr[index];  //Assuming no error checking is required.
+        }
```

Comment:
where is `index < 0` ?
if no error checking required then `arr[index]` would be enough